views/tutorial.ejs

```diff
@@ -11,28 +11,29 @@
   integrity="sha512-r3qr7vMdHYmWUzqC7l4H/W03ikAyiFIQld+B71yAmHhu7wcsnvm/adhikM+FzDxxK9ewznhCVnAm+yYZpPBnSg=="
   crossorigin="anonymous" />
 <script>
```

Comment:
There seems to be a missing `hljs.initHighlightingOnLoad()` here.

Reply:
I took it out because I thought it's not necessary. It only makes this part looks weird.
![image](https://user-images.githubusercontent.com/61715204/94361793-f3109280-00f1-11eb-82ba-3f95a58e57ba.png)
Other parts are still highlighted because of line 71.